projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -175,19 +175,14 @@ export function NewsletterAccessRadioButtons( {
 						  ]
 						: [] ),
 					{
-						label: `${ accessOptions.subscribers.label } (${ numberFormat( subscribersReach, {
-							notation: 'compact',
-							maximumFractionDigits: 1,
-						} ) })`,
+						label: `${ accessOptions.subscribers.label } (${ formatNumberCompact(
+							subscribersReach
+						) })`,
```

Comment:
Is `maximumFractionDigits: 1` missing here?

Reply:
Thanks for the review. 

It's technically not missing / or shouldn't. The default defined in `numberFormatCompact` is `maximumFractionDigits: 1` - so it's more a case of being redundantly provided elsewhere. It can be omitted.